src/modules/hip/hip_declarations_inline.hpp

```diff
@@ -213,169 +43,15 @@ color_convert_hip_batch(T* srcPtr, U* dstPtr, RppiColorConvertMode convert_mode,
-    std::string kernel_file  = "resize.cl";
-    std::string kernel_name = "resize_crop_mirror_batch";
-    get_kernel_name(kernel_name, tensor_info);
-
-    handle.AddKernel("", "", kernel_file , kernel_name, vld, vgd, "")(srcPtr,
-                                                                      dstPtr,
-                                                                      handle_obj->mem.mgpu.srcSize.height,
-                                                                      handle_obj->mem.mgpu.srcSize.width,
-                                                                      handle_obj->mem.mgpu.dstSize.height,
-                                                                      handle_obj->mem.mgpu.dstSize.width,
-                                                                      handle_obj->mem.mgpu.maxSrcSize.width,
-                                                                      handle_obj->mem.mgpu.maxDstSize.width,
-                                                                      handle_obj->mem.mgpu.uintArr[0].uintmem,
-                                                                      handle_obj->mem.mgpu.uintArr[1].uintmem,
-                                                                      handle_obj->mem.mgpu.uintArr[2].uintmem,
-                                                                      handle_obj->mem.mgpu.uintArr[3].uintmem,
-                                                                      handle_obj->mem.mgpu.uintArr[4].uintmem,
-                                                                      handle_obj->mem.mgpu.srcBatchIndex,
-                                                                      handle_obj->mem.mgpu.dstBatchIndex,
-                                                                      tensor_info._in_channels,
-                                                                      handle_obj->mem.mgpu.inc,
-                                                                      handle_obj->mem.mgpu.dstInc,
-                                                                      in_plnpkdind,
-                                                                      out_plnpkdind);
-
-    return RPP_SUCCESS;
-}
-
 /******************** non_linear_blend ********************/
 
 template <typename T, typename U>
 RppStatus
 non_linear_blend_hip_batch_tensor(T* srcPtr1, T* srcPtr2, U* dstPtr, rpp::Handle &handle, RPPTensorFunctionMetaData &tensor_info)
 {
     int in_plnpkdind = getplnpkdind(tensor_info._in_format), out_plnpkdind = getplnpkdind(tensor_info._out_format);
-    InitHandle *handle_obj = handle.GetInitHandle();
     Rpp32u max_height, max_width;
-    max_size(handle_obj->mem.mgpu.csrcSize.height, handle_obj->mem.mgpu.csrcSize.width, handle.GetBatchSize(), &max_height, &max_width);
+    max_size(handle.GetInitHandle()->mem.mgpu.csrcSize.height, handle.GetInitHandle()->mem.mgpu.csrcSize.width, handle.GetBatchSize(), &max_height, &max_width);
```

Comment:
what is the reason for this change? Previous version might be more efficient

Reply:
This was done to maintain uniformity for the hip functions. Almost all hip functions were using handle.GetInitHandle() except just a few that I made the change for. Shall check for performance and make a common change in a subsequent PR if there is difference.